modules/installation-about-restricted-network.adoc

```diff
@@ -37,6 +38,10 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-aws-installer-provisioned"]
 :ipi:
 endif::[]
+ifeval::["{context}" == "installing-ibm-cloud-restricted"]
+:ipi:
+endif::[]
+
```

Comment:
@jeffnowicki Line 56 makes reference to AWS makes reference to services that still require internet access. I would like to add content that is specific/relevant to IBM Cloud VPC. Any input on what would still require internet access. Thanks.